src/workerd/server/tests/python/py_wd_test.bzl

```diff
@@ -54,7 +54,7 @@ def py_wd_test(
             data = data,
             name = name_flag + "@",
             args = args,
-            size = size,
-            tags = tags + ["py_wd_test"],
+            # size = size,
```

Comment:
This is probably not meant to be committed right?

Reply:
No this is just how I run tests locally =)